src/CommonItilObject_Item.php

```diff
@@ -189,7 +189,7 @@ public function prepareInputForAdd($input)
      */
     protected static function displayItemAddForm(CommonITILObject|TicketRecurrent $obj, array $options = [])
     {
-        if (!($obj instanceof static::$itemtype_1)) {
+        if (!(is_a($obj, static::$itemtype_1))) {
```

Comment:
It seems that it was a valid usage, see https://www.php.net/manual/en/language.operators.type.php (`Example #5 Using instanceof with other variables`).

However, in this case, `is_a()` seems preferable to me.

Reply:
I didn't know it was legal, PHPStan doesn't think it is for some reason. Let's keep is_a then.